cli/top.go

```diff
@@ -31,7 +31,8 @@ func (top *TopCommand) Init(c *Cli) {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return top.runTop(args)
 		},
-		Example: topExamples(),
+		Example:            topExamples(),
+		DisableFlagParsing: true,
```

Comment:
Thanks a lot for this update. While for other commands like `create`, `run` and so on, we use `flagSet.SetInterspersed(false)`in 
```
func (cc *CreateCommand) addFlags() {
	flagSet := cc.cmd.Flags()
	flagSet.SetInterspersed(false)

	c := addCommonFlags(flagSet)
	cc.container = c
}
```
How about making pouchd use the unified way of `flagSet.SetInterspersed(false)`?

Reply:
I have finished.